dependency/src/main/java/de/dagere/peass/execution/gradle/GradleBuildfileEditor.java

```diff
@@ -102,19 +102,32 @@ private void editGradlefileContents(final File tempFolder, final GradleBuildfile
    private void addDependencies(final GradleBuildfileVisitor visitor) {
       JUnitVersions versions = testTransformer.getJUnitVersions();
       boolean isExcludeLog4j = testTransformer.getConfig().getExecutionConfig().isExcludeLog4jSlf4jImpl();
+      boolean isAnbox = testTransformer.getConfig().getExecutionConfig().isUseAnbox();
+      final String peass = " // Added dynamically by Peass.";
       if (visitor.getDependencyLine() != -1) {
          for (RequiredDependency dependency : RequiredDependency.getAll(versions)) {
             final String dependencyGradle;
             if (isExcludeLog4j && dependency.getMavenDependency().getArtifactId().contains("kopeme")) {
                String excludeString = "{ exclude group: '" + MavenPomUtil.LOG4J_GROUPID + "', module: '" + MavenPomUtil.LOG4J_SLF4J_IMPL_ARTIFACTID + "' }";
                dependencyGradle = "implementation ('" + dependency.getGradleDependency() + "') " + excludeString;
+            } else if (isAnbox && dependency.getGradleDependency().contains("kopeme")) {
```

Comment:
Is this really the intended if statement? Because this will only happen if log4j is not excluded - will this never happen, or is it just for current implementation convenience, that you do not handle the case of `isAnbox && isExcludeLog4j`? If the latter is true, I'd recommend to add a small comment that this case should be addressed in the future.

Reply:
I've added a comment for the future.